src/routing_node.rs

```diff
@@ -836,6 +840,86 @@ impl RoutingNode {
                                   refresh_our_own_group: bool) -> RoutingResult {
         unimplemented!()
     }
+
+    fn handle_cache_get(&mut self, request : ExternalRequest,
```

Comment:
Here we try to get the data direct from the lru_time cache, no need for authority etc.